src/hpilo_exporter/prometheus_metrics.py

```diff
@@ -16,6 +16,7 @@
 hpilo_network_gauge = Gauge('hpilo_network', 'HP iLO network status', ["product_name", "server_name"])
 hpilo_temperature_gauge = Gauge('hpilo_temperature', 'HP iLO temperature status', ["product_name", "server_name"])
 hpilo_firmware_version = Gauge('hpilo_firmware_version', 'HP iLO firmware version', ["product_name", "server_name"])
+hpilo_power_status_gauge = Gauge('hpilo_power_status', 'HP iLO Server Power Status', ["product_name", "server_name"])
```

Comment:
Please write the description in lower case just as the other descriptions.

Reply:
I have resolved this problem. please review.